zerver/lib/remote_server.py

```diff
@@ -82,7 +83,9 @@ class RealmDataForAnalytics(BaseModel):
     host: str
     url: str
     name: str = ""
+    description: str = ""
     org_type: int = 0
+    icon_url: str = ""
```

Comment:
not sure whether we want to send `icon_url` here or if we meant the actual image data - so went with the basic option first.